### PR TITLE
Suraj Sarangi is on the page

### PR DESCRIPTION
**Pull request by SurajSarangi**
Added my card to contribute to the repo. Hope it gets merged

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,7 +110,43 @@ <h2>A project for first-time contributions</h2>
...
+                </p>
+                <p class="about">
+                    Engineer || Front-end dev || Back-end dev || Data scientist
+                    <br><br>
+                    Find me on <a href="https://twitter.com/_suraj_sarangi" target="_blank">Twitter</a> or <a href="http://linkedin.com/in/surajsarangi" target="_blank">LinkedIN</a>
+                </p>
+                <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
```
Comment (lax0248917) [inline index.html:130]: Hello @SurajSarangi, thank you for contributing. There are some changes that need to be made to your index.html file. Please remove this blank line and be sure to run `npm run prettier-html` to insure that your file meets the styling requirements. Thank you.

**Review by lax0248917 — CHANGES_REQUESTED** [review]
Let us know if you have any questions. 👍

**Comment by SurajSarangi** [comment]
I removed the blank line as mentioned. There still seems to be conflicts however. Anything I can do? Thanks

**Comment by lax0248917** [comment]
Hello @SurajSarangi, you are all set. I went ahead and resolved any merge conflicts. Thank you for your contribution and good luck with open source in the future. Take care 🚀

**Pull request merged**